core/src/test/java/io/grpc/ClientInterceptorsTest.java

```diff
@@ -425,6 +425,21 @@ public void authorityIsDelegated() {
     assertEquals("auth", intercepted.authority());
   }
 
+  @Test
+  public void customOptionAccessible() {
+    CallOptions.Key<String> customOption = CallOptions.Key.of("custom", null);
+    
+    ClientInterceptor interceptor = spy(new NoopInterceptor());
+    Channel intercepted = ClientInterceptors.intercept(channel, interceptor);
+    CallOptions callOptions = CallOptions.DEFAULT.withOption(customOption, "value");
```

Comment:
Could you organize this test into setup, test and  verify sections, each separated by a new line?  It's hard to see what  is being tested at a glance.